index.html

```diff
@@ -30,7 +30,7 @@ <h4>Job title description</h4>
           
           <a class = "social-link" href="#"><img src="email.svg" alt="Email"/></a>
           <a class = "social-link" href="#"><img src="web.svg" alt="link"/></a>
-          <a class = "social-link" href="#"><img src="linkedin.svg" alt="LinkedIn"/></a>
+          <a class = "social-link" href="#"> <span id = "linkedin"><img src="linkedin.svg" alt="LinkedIn"/></span></a>
```

Comment:
spaces around equals in HTML are a nono; `class="social-link"` and `class="linkedin"`